libs/domains/organizations/feature/src/lib/helm-repository-create-edit-modal/helm-repository-create-edit-modal.tsx

```diff
@@ -160,13 +161,19 @@ export function HelmRepositoryCreateEditModal({
             name="url"
             control={methods.control}
             rules={{
-              pattern: new RegExp(
-                `^(${
-                  watchKind === 'HTTPS' ? 'http(s)' : 'oci'
-                }?:\\/\\/)[\\w.-]+(?:\\.[\\w\\.-]+)+[\\w\\-\\._~:/?#[\\]@!\\$&'\\(\\)\\*\\+,;=.]+$`,
-                'gm'
-              ),
               required: 'Please enter a repository url.',
+              validate: (input) => {
+                const regex = new RegExp(
+                  `^(${
+                    watchKind === 'HTTPS' ? 'http(s)' : 'oci'
+                  }?:\\/\\/)[\\w.-]+(?:\\.[\\w\\.-]+)+[\\w\\-\\._~:/?#[\\]@!\\$&'\\(\\)\\*\\+,;=.]+$`,
+                  'gm'
+                )
+                return (
+                  input?.match(regex) !== null ||
+                  `URL must start with «${watchKind === 'HTTPS' ? 'http(s)' : 'oci'}://»`
```

Comment:
Not correct because we need to have the end of the url `URL must start with «http(s)://» and finish with an extension`  

![Capture d’écran 2024-03-21 à 09 33 33](https://github.com/Qovery/console/assets/533928/10a78374-b49d-4ab6-bd0a-735bcdcb973b)

Reply:
Not fond of your proposal, does `URL must be valid and start with «http(s)://»` is ok for you?

Reply:
Yes it's good as well!

Reply:
Fixed https://github.com/Qovery/console/pull/1249/commits/e69a2dc2659e18c8781bb1ff44ae15f6d92fa0ce